src/HealthChecks.DynamoDb/DynamoDbOptions.cs

```diff
@@ -7,10 +9,13 @@ namespace HealthChecks.DynamoDb
     /// </summary>
     public class DynamoDBOptions

```

Comment:
```suggestion
    public class DynamoDbOptions
```

Just an oversight. It's a breaking change but I think not a big problem.

Reply:
I would prefer not to change this in this PR, as I haven't actually defined the name or touched this, could this be done as a separate PR instead?

Reply:
ok